doc/extensions/myexampleextension/azext_myexampleextension/__init__.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If the extension could be developed by the 3rd party, then I suggest we don't put in such detail, rather just some place-holder text